rllib/examples/models/batch_norm_model.py

```diff
@@ -13,13 +13,73 @@
 torch, nn = try_import_torch()
 
 
+class KerasBatchNormModel(TFModelV2):
```

Comment:
Confirmed working now right?

Reply:
Just moved it up in the file. It was always there. I rather have users use this model as it's more in line with our ModelV2 coding standards. Just for the PPO case, it doesn't work. However, our batch_norm example choses the correct model class based on the algo used (for PPO, it choses the old-style model).